### DISCUSSION TITLE
Pythia progress (or roadmap) discussion on the Pangeo weekly meeting?

### DISCUSSION THREAD

**Discussion by rsignell-usgs** [ProjectPythia/projectpythia.github.io#37] [General]
Would anyone be interesting in discussing Pythia progress (or roadmap) on the [Pangeo weekly meeting](https://docs.google.com/document/d/1BkL0arf1Lz6fHgVBEJNxKbFmVN8glNQXmDKdsuT0GcU/edit?usp=sharing) today?  Or if not today some other Wednesday?

**Comment by clyne** [comment]
I have a conflict with the afternoon meetings. Maybe someone else is available (@brian-rose , @dopplershift , @kmpaul ). Otherwise happy to next week at morning meeting.

**Comment by brian-rose** [comment]
Great idea, but I also have a conflict today.

I'd be happy to do so next Wednesday 1/27 if @clyne is unavailable.

**Comment by ktyle** [comment]
I was able to join in and give a brief overview of the project. Notes from the call are below:  

[Pangeo Weekly Check-in](https://docs.google.com/document/d/1BkL0arf1Lz6fHgVBEJNxKbFmVN8glNQXmDKdsuT0GcU) 

The attendees expressed interest in joining our regularly-scheduled Thursday meetings and asked if we could post links to them on the [Panego schedule.](https://pangeo.io/meeting-notes.html).

**Reply by clyne** [reply]
Thanks, Kevin!

**Reply by kmpaul** [reply]
Should we make a PR to the Pangeo website to update the meetings page?

**Reply by ktyle** [reply]
That would make the most sense ... I'd be happy to do so but have a celebratory hangout to attend shortly :)  @kmpaul / @clyne could you take the lead on this?

**Reply by kmpaul** [reply]
I can try to do it.

**Reply by rsignell-usgs** [reply]
Thanks @ktyle for joining the Pangeo check-in meeting yesterday!   I thought of Pythia when I was reviewing the excellent [office hours discussion](https://discourse.pangeo.io/t/pangeo-office-hours/669) on the Pangeo Discourse (that unfortunately petered out last June), particularly the suggested structure by @dcherian https://discourse.pangeo.io/t/pangeo-office-hours/669/2).

**Reply by kmpaul** [reply]
I've attempted a PR to add the meeting information to the Pangeo Meetings page.

https://github.com/pangeo-data/pangeo/pull/813
